### DISCUSSION TITLE
How to get noticed by companies using OSS?

### DISCUSSION THREAD

**Discussion by suitupstan** [EddieHubCommunity/support#3172] [Ideas] [answered]
How to get noticed by companies using OSS and get job?

Context: https://www.youtube.com/watch?v=wQvlaYc5nl8

CC: @schmelto

**Comment by eddiejaoude** [comment, marked as answer]
Some tips...

- Add different types of contributions/collaborations to one's resume https://www.youtube.com/watch?v=IuspU5OMis8
- Contribute to the companies you wish to work with
- Engage in their social posts and add value to the conversation

**Reply by kaiwalyakoparkar** [reply]
Thanks for the great tips @eddiejaoude :raised_hands:

**Reply by eddiejaoude** [reply]
Thanks for the great feedback!
